From c8d927e4d6b1d7a9a2787ac8db4d6c235f8bc224 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 15 Sep 2014 10:58:42 +0200
Subject: [PATCH 10/64] dns: send IPv6 connectivity probes to RIPE DNS rather
 than Google

Short of disabling the code (we may do that in a subsequent patch
anyway), avoid sending to Google.
---
 net/dns/host_resolver_impl.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/net/dns/host_resolver_impl.cc b/net/dns/host_resolver_impl.cc
index 2584d8f16008..9de11dd34bff 100644
--- a/net/dns/host_resolver_impl.cc
+++ b/net/dns/host_resolver_impl.cc
@@ -107,10 +107,10 @@ const unsigned kMinimumTTLSeconds = kCacheEntryTTLSeconds;
 // cached.
 const int kIPv6ProbePeriodMs = 1000;
 
-// Google DNS address used for IPv6 probes.
+/* RIPE NCC k.root-servers.net. 2001:7fd::1 (anycasted) */
 const uint8_t kIPv6ProbeAddress[] =
-    { 0x20, 0x01, 0x48, 0x60, 0x48, 0x60, 0x00, 0x00,
-      0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x88, 0x88 };
+    { 0x20, 0x01, 0x07, 0xfd, 0x00, 0x00, 0x00, 0x00,
+      0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01 };
 
 // We use a separate histogram name for each platform to facilitate the
 // display of error codes by their symbolic name (since each platform has
-- 
2.15.0

